x-pack/plugins/fleet/server/services/package_policy.ts

```diff
@@ -944,7 +944,7 @@ export function overridePackageInputs(
     // If there's no corresponding input on the original package policy, just
     // take the override value from the new package as-is. This case typically
     // occurs when inputs or package policies are added/removed between versions.
-    if (!originalInput) {
+    if (originalInput === undefined) {
```

Comment:
Do we really need that change?

Reply:
This fixed typescript errors later in the function since we explicitly narrow the type to exclude `undefined` here. It removed the need for `?` or `!` in some of the checks below.